zitadel/v2/helper/client.go

```diff
@@ -44,12 +44,12 @@ func GetClientInfo(insecure bool, domain string, token string, jwtProfileFile st
 		return nil, fmt.Errorf("either 'jwt_profile_file' or 'jwt_profile_json' is required")
 	}
 
-	issuer := ""
-	if port != "" {
+	issuer := domain
+	//issuer only contains the port if it's not the default port of the schema
+	if port != "" && !(insecure && port == "80") && !(!insecure && port == "443") {
```

Comment:
can we make it more readable? something like...

```go
scheme := "https://"
if insecure {
    options = append(options, zitadel.WithInsecure())
    scheme = "http://"
}

if port == "80" && insecure || port == "443" && !insecure {
    port = ""
}

issuer := scheme + domain + ":" + port
```

Reply:
@eliobischof can you please make the change yourself and add me as reviewer as stebenz is in holiday for the next two weeks?

Reply:
done